website/docs/fun/counting_v2.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Counting System
+---
+
+This command controls the counting game and the statistics command.
+
+This version of a counting command allows for the usage of base 10 and roman numerals, optionally joined by the usage of math functions and/or operators to count from 1, incrementing by 1, until failure.
+
+Also included are statistics for the server and each user for how many times they have counted and how many times they've been correct.
```

Comment:
You can drop this sentence after changing the first line of the documentation as suggested earlier.